scripts/Tools/xmlquery

```diff
@@ -190,10 +190,14 @@ formatter_class=argparse.RawDescriptionHelpFormatter)
         args.type, args.valid_values, args.partial_match, args.file
 
 def get_value_as_string(case, var, attribute=None, resolved=False, subgroup=None):
-    thistype = case.get_type_info(var)
-    value = case.get_value(var, attribute=attribute, resolved=resolved, subgroup=subgroup)
-    if value is not None and thistype:
-        value = convert_to_string(value, thistype, var)
+    if var in ["THREAD_COUNT", "TOTAL_TASKS", "TASKS_PER_NODE", "NUM_NODES", "SPARE_NODES", "TASKS_PER_NUMA", "CORES_PER_TASK"]:
```

Comment:
I wonder if there should be a message attached to the output to the effect that these variables cannot be modified with xmlchange?

Reply:
@jedwards4b , i'm not sure how to do that without cluttering the output in the case that the user wants the raw value.

Reply:
Can you use logger.info and suppress if --value is used?

Reply:
@jedwards4b I have an idea.